### PR TITLE
PSMDB-1103 Release notes 6.0.0-1

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

**Review by igorsol — APPROVED** [review]
LGTM

File: source/release_notes/6.0.2.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+:Installation: `Installing Percona Server for MongoDB <https://www.percona.com/doc/percona-server-for-mongodb/6.0/install/index.html>`_
+
+
+We are pleased to announce the availability of Percona Server for MongoDB (PSMDB) 6.0.2-1 – the new major version of the source available, drop-in replacement of MongoDB 6.0 Community edition. It is available for `download from Percona website <https://www.percona.com/downloads/percona-server-mongodb-6.0/#>`_ and for `installation from Percona Software Repositories <https://www.percona.com/doc/percona-server-for-mongodb/6.0/install/index.html>`_.
+
+PSMDB 6.0.2-1 fully supports MongoDB 6.0 protocols and drivers and does not require any code modifications.
+
+PSMDB 6.0.2-1 includes all the features of MongoDB 6.0 Community Edition. These include enhancements to the existing features, plus the bunch of new ones. 
+
+Among new features are the following:
```
Comment (janwieremjewicz) [inline source/release_notes/6.0.2.rst]: I do not see anything about queryable encryption :( and the basic mode (explicit) is AFAIK included in CE

**Review by sandraromanchenko — APPROVED** [review]

**Comment by kwilktx** [comment]
Make the  required change to the new mongosh shell more visible.

==> Change this:

Percona Server for MongoDB packages now include mongosh instead of mongo CLI. If you [install Percona Server for MongoDB from tarballs](https://psmdb-6-0-preview.onrender.com/install/tarball.html#tarball), you must install mongosh from a separate tarball.

to this ==>

Percona Server for MongoDB packages now include mongosh instead of the previous mongo shell. The previous legacy mongo shell was deprecated in MongoDB 5.0 and has been removed in version 6.0. Some older methods are unavailable or have been replaced with newer ones for the new mongosh. Please review any methods for compatibility. 

If you [install Percona Server for MongoDB from tarballs](https://psmdb-6-0-preview.onrender.com/install/tarball.html#tarball), you must install mongosh from a separate tarball.

==> More to come

File: source/release_notes/6.0.2.rst (hunk 1 above)
Comment (nastena1606) [reply source/release_notes/6.0.2.rst]: It's been decided not to include it in the RN

**Pull request merged**